src/mongocrypt-kms-ctx.c

```diff
@@ -1228,12 +1384,21 @@ _mongocrypt_kms_ctx_init_gcp_auth (
    bool ret = false;
    ctx_with_status_t ctx_with_status;
 
+   if (!kms) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.